lib/generators/index.js

```diff
@@ -3,26 +3,31 @@ const Chance = require('chance').Chance();
 const Format = require('./format');
 const Randexp = require('randexp').randexp;
 
-const mock = schema => {
+const mock = (schema, useExample)  => {
     let mock;
     if (schema) {
         let type = schema.type || findType(schema);
+        let example = schema.examples || schema.example;
         /**
          * Get the mock generator from the `type` of the schema
          */
-        const generator = Generators[type];
-        if (generator) {
-            mock = generator.call(null, schema);
+        if (example && useExample) {
```

Comment:
We should probably use `if (typeof example !== 'undefined' && useExample)` here instead. I have some boolean property examples as `false` and that's causing this to generate random values (sometimes true!) for these instead.